test/quic_ackm_test.c

```diff
@@ -945,7 +945,7 @@ static int test_rx_ack_actual(int tidx, int space)
             if (!TEST_ptr(ack))
                 goto err;
 
-            if (!TEST_size_t_eq(ack->num_ack_ranges, s->num_ack_ranges))
+            if (!TEST_uint64_t_eq(ack->num_ack_ranges, s->num_ack_ranges))
```

Comment:
`size_t`

Reply:
Okie

Reply:
I hope you understand that I'll pay a little extra attention to what happens with the fixup that does these `unint64_t` → `size_t` reversals...  I do recall build failures, but maybe I over-reacted.  I guess we'll see.